largest_num.js

```diff
@@ -31,3 +31,40 @@ function stringArgument() {
 }
 const array = ['do','go','come'];
 console.log(stringArgument(array));
+
+//Exercise(6) number one(1)
+//Write a function that takes an array of strings and returns a new array with all the strings capitalized.
+// -Define an array of strings (exercise(5) number one(1))
+// -Loop through with arguments
+//- Initialize a variable to hold an Uppercase method
+// -Finally return the result with a map method to hold the variable
+
+function stringCapitalized() {
+    for (let valueOfStrings of arguments) {
+      const capitalized = name => name.toUpperCase();
+    return valueOfStrings.map(capitalized);
+  }
+}
+console.log(stringCapitalized(array));
+
+//Exercise(6) number two(2)
+//Write a function that takes a string as an argument and returns the number of vowels in the string.
+// -Define string
+// -Initialized a count variable to zero(0)
+// -Loop through the string
+// -Initialized a variable to hold the string with lowerCase method 
+// -An IF statements to compare vowels
+// -Assign count +1
+// -Finally return count
+
+const sentence = 'my name is somebody';
+function arrayOfVowels(numOfVowels) {
```

Comment:
Please consider naming your functions. They should make a meaning to what it will be used for. 
e.g `function arrayOfVowels(numOfVowels)` can be `function numberOfVowels(word)`
and the parameter passed to the function is also not a good one. it could be str/word/sentence or something more meaningful.